saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/credentials/Saml2X509Credential.java

```diff
@@ -45,6 +45,44 @@
 	private final X509Certificate certificate;
 	private final Set<Saml2X509CredentialType> credentialTypes;
 
+	/**
+	 * Creates a Saml2X509Credentials representing Identity Provider credentials
```

Comment:
I don't think it's entirely true that encryption credentials are for the Identity Provider. Or, while that's certainly the common case, I think it makes more sense to affirm that in something that has more context like `RelyingPartyRegistration`.

Would you please change this to something like:

```
Create a {@link Saml2X509Credential} that can be used for encryption.

@param certificate the certificate to use for encryption
```

Reply:
Also, based on the [Code Style guidelines](https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting) would you please use the imperative ("Create" instead of "Creates") in your JavaDoc?

Reply:
@jzheaux I updated the JavaDoc according to your comments and the code style guidelines.